qualtran/protos/data_types.proto

```diff
@@ -45,3 +47,15 @@ message QFxp {
     optional IntOrSympy num_frac = 2;
     optional bool signed = 3;
 }
+
+message QDataType {
+  oneof val {
+    QBit qbit = 1;
+    QAny qany = 2;
+    QInt qint = 3;
+    QIntOnesComp qintoc = 4;
```

Comment:
maybe have `qint_ones_comp` to match the type?

Reply:
Done.